typescript/src/merchi.ts

```diff
@@ -229,12 +229,12 @@ export class Merchi {
     return apiFetch(resource, options);
   };
 
-  public getCurrentUser = () => {
+  public getCurrentUser = (embed?: any) => {
     if (!this.sessionToken) {
       return Promise.resolve(null);
     }
     return this.Session.get(this.sessionToken, {
-      embed: { user: { enrolledDomains: {domain: {}} } }
+      embed: embed ? embed : { user: { enrolledDomains: {domain: {}} } }
```

Comment:
this breaks the line below in the case that the caller provides an embed descriptor not including `user`.

the provided embed descriptor should be merged with the minimum requirement, it should not replace the minimum requirement

Reply:
I would have thought to do that. I would have thought that if I provided an embed WYSIWYG. But I like your idea of having a minimum embed.